inc/class-astra-dynamic-css.php

```diff
@@ -308,8 +308,24 @@ public static function return_output( $dynamic_css, $dynamic_css_filtered = '' )
 			// check the selection color incase of empty/no theme color.
 			$selection_text_color = ( 'transparent' === $highlight_theme_color ) ? '' : $highlight_theme_color;
 
+			// Global Palette Css for :root.
+			$global_palette = astra_get_option( 'global-color-palette' );
+		
+			$respective_palette = $global_palette[ $global_palette['patterntype'] ];
+								
+			$finalpalette = array();
+			
+			foreach ( $respective_palette as $key => $value ) {
+				if ( $value[0] ) {
+					$palette_variables                  = '--ast-global-palette' . $key;
```

Comment:
```suggestion
					$palette_variables                  = '--ast-global-palette-' . $key;
```

Let's add an extra `-` before the dynamic part of the string, You will need to update it everywhere else as well.